CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Changelog
 
 
+## 1.0.2
```

Comment:
Should we release this right away? Are the APIs officially support this feature? If so, I'd consider this a minor update instead of a patch.